RUFAS/routines/manure/manure_treatments/manure_treatment_daily_output.py

```diff
@@ -296,16 +301,18 @@ class ManureTreatmentDailyOutput(LiquidManurePortionProtocol):
     solid_manure_daily_mass: float = 0.0
     solid_manure_daily_mass_unit: MeasurementUnits = MeasurementUnits.KILOGRAMS
 
-    # different unit from docMeasurementUnitsing m^3 vs m^3/day?
-    biogas: float = 0.0  # biogas production per day (m3/day)
-    biogas_unit: MeasurementUnits = MeasurementUnits.CUBIC_METERS_PER_DAY
+    methane_generation_mass: float = 0.0
+    methane_generation_mass_unit: MeasurementUnits = MeasurementUnits.KILOGRAMS_PER_DAY
 
-    biogas_energy_content: float = 0.0  # biogas energy content (MJ/m3)
-    biogas_energy_content_unit: MeasurementUnits = MeasurementUnits.MEGAJOULES_PER_CUBIC_METER
+    methane_energy_content: float = 0.0
+    methane_energy_content_unit: MeasurementUnits = MeasurementUnits.MEGAJOULES
```

Comment:
```suggestion
   biogas: float = 0.0
   biogas_unit: MeasurementUnits = MeasurementUnits.KILOGRAMS_PER_DAY
```
Reverting output name changes to maintain backwards compatibility